### ISSUE TITLE
add new units

### ISSUE THREAD

**Issue by bequrios** [zazuko/cube-creator#1484]
For a BAFU project, we need the following units

- Megawatthours per capita per year
- Kilowatthours per square metre per year
- tons per capita per year
- Kilograms per square metre per year

Thanks

**Comment by Rdataflow** [comment]
@bequrios Lindas harvests units from upstream qudt directly. The best way to have new units in Lindas is to open a PR upstream at https://github.com/qudt/qudt-public-repo/ so it will be reusable (once only paradigm)...

cc @MartinaElsawy @CDiGallo

**Comment by l00mi** [comment]
@Rdataflow I full agree, but this might take quite a while to get done. To deblock projects, we did in parallel also add units here: https://gitlab.ldbar.ch/pipelines/lindas-metadata/-/tree/develop/units?ref_type=heads

**Comment by Rdataflow** [comment]
@l00mi we already collected good experience in the case of a PR made by @kronmar ... a few weeks later the patched release was harvested on Lindas 👍

**Comment by bequrios** [comment]
ok, i will try to do it on the upstream repo.

**Comment by MartinaElsawy** [comment]
Hi Benedikt, great, let us try what Tom and Thomas suggestested (thanks Guys!) and in case of no success. Please open an issue in the official BAFU board for the Cube Creator project and add it to the "upcoming sprint" row (or label it, same same) so we can check with Claudio/Sabine for treatment.

**Comment by tpluscode** [comment]
Closing for now as this should be out of scope. Please reopen if necessary
